src/api/API.js

```diff
@@ -274,9 +274,9 @@ const API = {
    * @returns {Promise}
    * @throws {APIError} On server error.
    */
-  async removeUsers(groupId, userIds) {
+  async removeUser(groupId, userId) {
     return axios
-      .delete(`/groups/${groupId}/removeUsers`, { data: { users: userIds } })
+      .delete(`/groups/${groupId}/removeUser`, { data: { user: userId } })
       .then(response => response.data);
```

Comment:
Just need to make sure you update the jsdoc comment for the `userId` param

Reply:
Good catch